start.sh

```diff
@@ -42,6 +42,20 @@ prep() {
     ${COMPOSE} pull
 }
 
+# Gets the IP of the local machine.
+getIp() {
+    HOSTNAME_VAL="$(hostname -i)"
```

Comment:
Wasn't the original point of this PR that you couldn't rely on `-i` being available?